libmscore/bracket.cpp

```diff
@@ -341,15 +341,8 @@ Element* Bracket::drop(const DropData& data)
       Element* e = data.element;
       if (e->type() == Element::Type::BRACKET) {
             Bracket* b = static_cast<Bracket*>(e);
-            b->setParent(parent());
-            b->setTrack(track());
-            b->setSpan(span());
-            b->setFirstStaff(firstStaff());
-            b->setLastStaff(lastStaff());
-            b->setLevel(level());
-            score()->undoRemoveElement(this);
-            score()->undoAddElement(b);
-            return b;
+            score()->undoChangeBracketType(this, b->bracketType());
+            return this;
```

Comment:
I have one doubt: is the code leaking memory here since "e" is not deleted before "return this" (as I have rewritten this part of code, "e" is no more necessary after the change of bracket type)? I see that when the type is not changed "e" is deleted.